src/templates/podcast.js

```diff
@@ -23,10 +24,6 @@ const PODCAST_BREADCRUMB = [
         target: '/',
     },
     { label: 'Learn', target: '/learn' },
-    {
-        label: 'Podcasts',
-        target: '/podcasts',
```

Comment:
Ah I meant we should keep this breadcrumb "Podcast" but the `target` should be the current page instead of `/podcasts`